backends/events/sdl/sdl-events.cpp

```diff
@@ -900,8 +938,10 @@ bool SdlEventSource::remapKey(SDL_Event &ev, Common::Event &event) {
 void SdlEventSource::resetKeyboardEmulation(int16 x_max, int16 y_max) {
 	_km.x_max = x_max;
 	_km.y_max = y_max;
-	_km.delay_time = 25;
+	_km.delay_time = 12;
```

Comment:
Original code updates joystick mouse at 40 fps which looks very rough to the eye. That was just  too slow. New code updates at 60 fps, which looks smooth to the eye. It is a visual difference like day and night. However it does make the mouse pointer twice as fast. I will fix this in next commit.